### DISCUSSION TITLE
Something wrong?

### DISCUSSION THREAD

**Discussion by ZatoichiThe** [imartinez/privateGPT#1839] [General]
I tried various shorter and longer pdf documents, and find both the ollama version (mistral) and the OpenAi verison relly bad at getting information from the document. I tried short stories from Roald Dahl - app 250 pages. the first one is called "Taste". Itdoesnt know the story, the title, how many stories there are, tells me there is only one story, than that there are multiple. etc. It cant give useful answers to the content.
I tried Austrian laws, it cant give me information on the different articles, besides very very general information on what the document is about in whole.
I tried a table with theatrical admissions from cinemas, it cant extract useful information out of it, etc.

I think it is the embedding, and not the settings (halluzination and stuff). Has anyone better experience with this?

**Comment by JTMarsh556** [comment]
ollama didn't do well in the last version or this version for me.  I have been using llama because of that.
